### DISCUSSION TITLE
wave watch 3 cases

### DISCUSSION THREAD

**Discussion by JMSnyder1980** [NOAA-EMC/WW3#790] [General]
Are there any test cases for ww3 v6.17?

**Comment by JessicaMeixner-NOAA** [comment]
The regression tests for 6.07 are a great place to look for a variety of different test cases.  Section 5.11 "Running Test Cases" in the [manual](https://github.com/NOAA-EMC/WW3/wiki/Manual) has a list of the test cases and quick description which could help you decide which one to start with.
